pom.xml

```diff
@@ -17,59 +17,248 @@
     </modules>
 
     <properties>
+        <!-- std -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.version>1.8</java.version>
+
+        <!-- spring -->
+        <spring-core.version>4.2.3.RELEASE</spring-core.version>
+        <sprint-boot.version>1.3.0.RELEASE</sprint-boot.version>
+
+        <!-- raspberry pi -->
+        <pi4j.version>0.0.5</pi4j.version>
+
+        <!-- logging -->
         <slf4j.version>1.7.7</slf4j.version>
-        <junit.version>4.11</junit.version>
         <logback.version>1.1.2</logback.version>
+
+        <!-- testing -->
+        <junit.version>4.12</junit.version>
+        <assertj.version>3.2.0</assertj.version>
+        <mockito.version>1.10.19</mockito.version>
+
+        <!-- other -->
+        <wagon.version>2.6</wagon.version>
+
+        <!-- plugins -->
+        <jaxb2-plugin.version>0.12.3</jaxb2-plugin.version>
+        <compiler-plugin.version>3.1</compiler-plugin.version>
+        <assembly-plugin.version>2.6</assembly-plugin.version>
+        <wagon-plugin.version>1.0</wagon-plugin.version>
+        <exec-plugin.version>1.3.2</exec-plugin.version>
+        <coveralls-plugin.version>2.2.0</coveralls-plugin.version>
+        <jacoco-plugin.version>0.7.1.201405082137</jacoco-plugin.version>
     </properties>
 
+    <dependencyManagement>
```

Comment:
This is where you will specify all the dependencies you want to use in the project and what version you will be using.